src/current/v24.1/cutover-replication.md

```diff
@@ -154,16 +154,115 @@ At this point, the primary and standby clusters are entirely independent. You wi
 
 ## Cut back to the primary cluster
 
-After cutting over to the standby cluster, you may need to move back to the original primary cluster, or a completely different cluster. This process is manual and requires starting a new replication stream.
+After cutting over to the standby cluster, you may need to move back to the original primary cluster to serve your application.
```

Comment:
I'd lean into either "cut back" or "move back", but it's a little confusing to see both.